packages/rendermime/style/base.css

```diff
@@ -22,6 +22,7 @@
   border: none;
   margin: 0px;
   padding: 0px;
+  line-height: normal;
```

Comment:
This change is a separate issue. Can you remove it from this PR?

Reply:
(Looks like this CSS change is from #7832.)

Reply:
Yes, not sure how it happened.

Reply:
> (Looks like this CSS change is from #7832.)

I created branch out of my previous branch:) now I removed and checked in. Kindly review.